examples/tree-view/actions/index.js

```diff
@@ -0,0 +1,26 @@
+export const INCREMENT = 'INCREMENT'
+export const CREATE_NODE = 'CREATE_NODE'
+export const ADD_CHILD = 'ADD_CHILD'
+
+export function increment(nodeId) {
+  return {
+    type: INCREMENT,
+    nodeId
+  }
+}
+
+let nextId = 0
+export function createNode() {
+  return {
+    type: CREATE_NODE,
+    nodeId: `new_${nextId++}`
+  }
+}
+
+export function addChild(nodeId, childId) {
```

Comment:
I don't get it, I was thinking about something like this:

```js
export function createNode() {
  return {
    type: CREATE_NODE,
    nodeId: generateId()
  }
}

export function addChild(nodeId) {
  return {
    type: ADD_CHILD,
    nodeId,
    childId: generateId()
  }
}

let nextId = 0
function generateId() {
  return `new_${nextId++}`
}
```

How does it make the reducers more complex?

Sorry if I'm missing something :smile: